src/index.js

```diff
@@ -3,4 +3,14 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import registerServiceWorker from './registerServiceWorker';
 
+const App = () => {
+    return (
+        <<h1>Hello</h1>
```

Comment:
Notice that you have two open tags "<<" in front of your h1, you'll want to remove one of those.